src/rt/backtrace/elf.d

```diff
@@ -58,6 +58,62 @@ struct Image
 
         return null;
     }
+
+    @property size_t baseAddress()
```

Comment:
This is an awful lot of logic for a property, and would be run needlessly if called multiple times by a caller that isn't aware of the implementation.  The baseAddress shouldn't change during the lifetime of an `Image`, so wouldn't it be better to put this in the constructor, and store the result as a field?